dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/AbstractMongoDBRdfConfigurationBuilder.java

```diff
@@ -182,8 +167,7 @@ private C getConf(C conf) {
             conf.setMongoPassword(pass);
         }
         conf.setMongoDBName(mongoDBName);
-        conf.setRyaInstanceName(mongoCollectionPrefix);
-        conf.setTablePrefix(mongoCollectionPrefix);
+        conf.setRyaInstanceName(mongoDBName);
```

Comment:
If Mongo DB Name and Rya Instance Name are always the same, why not just have one of those fields?